packages/lab-extension/style/base.css

```
@@ -9,3 +9,7 @@
   align-items: center;
   justify-content: center;
 }
+
+.jp-InterfaceSwitcher .lm-MenuBar-itemIcon {
```

Something like this might do the trick?

```suggestion
.jp-InterfaceSwitcher .lm-MenuBar-itemIcon svg {
```

---

Yeah you're right, it does render differently when it's a more specific selector in Firefox.